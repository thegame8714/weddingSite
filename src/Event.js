import React from 'react'
import { FormattedMessage } from 'react-intl'
import messages from './translation/sentences'

class Event extends React.Component {
  render() {
    return (
      <div>
        <h1>
          <FormattedMessage {...messages.event_title} />
        </h1>
        <ul className="event-list">
          <li>
            17:00 <FormattedMessage {...messages.guests_arrive} />
          </li>
          <li>
            17:30 <FormattedMessage {...messages.wedding_starts} />
          </li>
          <li>
            18:00 <FormattedMessage {...messages.aperitif} />
          </li>
          <li>
            19:00 <FormattedMessage {...messages.dinner} />
          </li>
          <li>
            21:00 <FormattedMessage {...messages.music} />
          </li>
        </ul>
      </div>
    )
  }
}

export default Event
