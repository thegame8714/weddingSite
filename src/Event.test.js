import renderer from 'react-test-renderer'
import React from 'react'
import Event from './Event'

describe('Event component', () => {
  it('should render the event page', () => {
    const tree = renderer.create(<Event />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
