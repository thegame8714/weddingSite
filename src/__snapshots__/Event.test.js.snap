// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event component should render the event page 1`] = `
<div>
  <h1>
    Event
  </h1>
  <ul>
    <li>
      - 17:00 Guests arrive
    </li>
    <li>
      - 17:30 Wedding starts
    </li>
    <li>
      - 18:00 Aperitif and snacks
    </li>
    <li>
      - 19:00 Dinner
    </li>
    <li>
      - 21:00 Music and dance until night
    </li>
  </ul>
</div>
`;
